### ISSUE TITLE
Proposal: Celadon & Purple Colour Scheme

### ISSUE THREAD

**Issue by torres126** [Automattic/wp-calypso#30541]
#### Introduction

Personally, I think it'd be great if a new colour scheme was offered which strayed away from the same blue (or black) which currently is being used, but still remains familiar. In terms of accessibility and general look, I've been really fond of celadon and hot purple. 

I'm not too certain if it's even possible to propose a new colour scheme (I'd be happy to work on this if it's just a matter of not having enough time to add more - I looked into this quite a bit for this issue), but this is something I've wanted to see for a while now, so figured I may as well make an issue. If it isn't, feel free to close. 

#### Possible Design

Primary: Celadon 500 
Accent: Hot Purple 500
Neutrals: Probably a darker grey - not certain

A quick concept, switching only accent and primary variables:

![gdfsdsfgsdgfgsdffdgssdfg](https://user-images.githubusercontent.com/43215253/52124137-2ac6ee80-2620-11e9-9c29-ae1483c781c3.png)

![sgdfsdgfsdfgdsfg](https://user-images.githubusercontent.com/43215253/52129637-ec393000-262f-11e9-822b-f9343d1cd444.png)

![gfdsfsgddfsgfsgd](https://user-images.githubusercontent.com/43215253/52129645-ef342080-262f-11e9-9f5b-eabab926f35f.png)

![gsdfgdfsdsfgdfg](https://user-images.githubusercontent.com/43215253/52129890-a2047e80-2630-11e9-891c-8420b5e36fd8.png)

![gdfgdfgfddfg](https://user-images.githubusercontent.com/43215253/52129881-9dd86100-2630-11e9-9b99-ea740944fca8.png)

#### Accessibility

Celadon and hot purple works well in sticking to accessibility standards whilst also looking pleasant. 

White on proposed primary - 5.63:1 (https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=007565)
White on proposed accent - 5.59:1 (https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=8051B4)

#### Naming

Stumped here. The first things which come to mind with purple is royalty (and rain & Prince!). Celadon is a bit more tricky, but I thought water/ocean. So maybe something like Royal Moana? 

#### Reasoning

See here: https://en.blog.wordpress.com/2019/01/08/customize-your-wordpress-com-dashboard/

Most of the more negative feedback towards the new schemes wishes to see a less "monotonous" colour scheme, despite the accessibility advantages. I feel like celadon and purple would offer this alternative, but also sticking to accessibility guidelines. Some comments blatantly suggest a green-like scheme as well. Some of many:

> I find it refreshing but would like to see more options. Orange is always a good color because it represents warmth and happiness. Green, also is a good color because it represents newness and growth.

> Oh dear! I don’t think these changes are very good, especially given that the classic blue is nowhere near like it used to be. Like many others have said, the new colours are too dark. Brighter colours would not only stand out a lot more but would also be very eye-catching. I know there are more colours on the way, but can we have some bright, cheerful colours, please?

> A most welcome improvement, thank you. I definitely prefer the higher contrast, so much easier to read. The “new” blue is darker which I prefer over the “old” blue. The added pink is a nice touch. I always use the classic editor and classic Stats, Comments, etc. so don’t see the “new” color scheme except when visiting the Reader and when I log in. And, yes, please give us more color choices in the near future, green would be lovely.

And truth be told, I just really wanted to see a similar colour scheme to this because I think it looks nice. :) 

(cc @drw158, @flootr)

**Comment by drw158** [comment]
@torres126 I'm so excited that you're excited about color schemes, even enough to make your own! I really appreciate all of the help you've given us on this project. We actually have someone working on more color schemes _right now_. Your exact scheme is not on our list, but I think there's one that uses Celadon that you'll really like.

**Comment by torres126** [comment]
Awesome, thanks @drw158! :)
